### ISSUE TITLE
SUGGESTION: Allay Support

### ISSUE THREAD

**Issue by Th3OneAndOnly** [minity-script/minity#2]
Allay (at [DoubleF3lix/Allay](https://github.com/DoubleF3lix/Allay)) is a tool used to convert a descriptive, rich-text like text format into minecraft text components. It would be super cool if we could write Allay-compatible strings in Minity and convert them on-the-way using Allay as an alternative to the json format.

**Comment by zocky** [comment]
This sounds interesting, though we already have html-like markup for the same thing. Any suggestions for how we should go about it?

Markdown-style formats are harder to parse unambiguosly as a part of syntax than XML, so this should either have a very obvious delimiter, or maybe allay files should be importable into `?constants`.

**Comment by Th3OneAndOnly** [comment]
That's right. Perhaps the syntax could introduce a new string-like multiline delimiter, something like:

```
?my_text = allay `
Hello [Beautiful](italic bold) World!
`
```

Another issue I thought of is transmitting errors in the Allay string to Minity users. If the Allay string contains errors we would have to catch the exception ([Allay/parser.py at main · DoubleF3lix/Allay (github.com)](https://github.com/DoubleF3lix/Allay/blob/main/allay/parser.py#L44)) and handle it. Unfortunately it looks like the information for the type of error is contained within the error message. Might have to open an issue because that is annoying to use from an API perspective.

**Comment by zocky** [comment]
I've never done any python<->JS API calls, so if you have any pointers, that'd be great
